config/application.yml

```diff
@@ -0,0 +1,7 @@
+api_key: 'key-4dabe8bf9b37373b4d042f81a649177a'
+domain: 'sandbox5904b917a0744e1fb224ff924e374218.mailgun.org'
+username: 'postmaster@sandbox5904b917a0744e1fb224ff924e374218.mailgun.org'
+password: 'fc205a5e07a8ce848b0d87917314e0d9'
+
+gmail_username: 'alex.magana@andela.com'
+gmail_password: 'asd!@#6353aMgx'
```

Comment:
This file should be git-ignored. I assume you're using **figaro** so next time, run:
``` shell
$ bundle exec figaro install
```
This creates the file and adds it to `.gitignore` file

Since you are already tracking this file, you can run:
``` shell
$ git rm --cached config/application.yml
```
to remove it from git